### PR TITLE
Fix dropout error in Bidirectional layer

### PR DESCRIPTION
**Pull request by joelthchao**
For issue #5940 and #5975 
`Bidirectional` wrapper produces error with any `Recurrent` layer with `dropout` or `recurrent_dropout` argument.

### CODE REVIEW

File: keras/layers/wrappers.py [hunk 1]
```diff
@@ -208,21 +208,30 @@ def compute_output_shape(self, input_shape):
         elif self.merge_mode is None:
             return [self.forward_layer.compute_output_shape(input_shape)] * 2
 
-    def call(self, inputs, mask=None):
-        y = self.forward_layer.call(inputs, mask)
-        y_rev = self.backward_layer.call(inputs, mask)
+    def call(self, inputs, training=None, mask=None):
+        y = self.forward_layer.call(inputs, mask=mask, training=training)
```
Comment (fchollet) [inline keras/layers/wrappers.py]: Better to check whether the layer's call method accepts the `training` and `mask` arguments.

**Comment by fchollet** [comment]
Also please add a unit test for Bidirectional with dropout (together with other Bidirectional tests), since this is clearly not covered.

**Comment by joelthchao** [comment]
Python2 issue, will fix it later.
